docusaurus.config.js

```diff
@@ -200,7 +200,7 @@ module.exports = {
                 {
                   label: 'Stronghold',
                   sublabel: 'Handle secrets securely',
-                  to: 'stronghold.rs/welcome',
+                  to: 'stronghold.rs/getting_started',
```

Comment:
This is actually why I added a welcome page in our docs-template. Maybe we should stay consistent here?

Will merge it for now. Let's discuss it again tho